bottlerocket-settings-models/settings-extensions/kubelet-device-plugin/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+//! Settings related to Amazon ECS
+use bottlerocket_model_derive::model;
+use bottlerocket_modeled_types::NvidiaDevicePluginSettings;
+use bottlerocket_settings_sdk::{GenerateResult, SettingsModel};
+use std::convert::Infallible;
+
+#[model(impl_default = true)]
+pub struct KubeletDevicePluginV1 {
```

Comment:
I'd prefer to expose this as `settings.kubelet-device-plugins` for consistency with our other plural settings clusters. So the type name should also be plural.

```suggestion
pub struct KubeletDevicePluginsV1 {
```